src/common/storage/backends/StorageClient.js

```diff
@@ -1,14 +1,13 @@
 /* globals define*/
 define([
     'client/logger',
-    'deepforge/gmeConfig'
+    'deepforge/gmeConfig',
+    'deepforge/StorageHelpers'
```

Comment:
If this is going to be refactored, `StorageHelpers` should be in `deepforge/storage`.